doc-content/enterprise-only/openshift/rhdm76-authoring-ha-ref.adoc

```diff
@@ -74,8 +74,11 @@ https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/
 |`DECISION_CENTRAL_MAVEN_PASSWORD` | `KIE_MAVEN_PWD` | Password to access the Maven service hosted by Decision Central inside EAP. | -- | True
 |`GIT_HOOKS_DIR` | `GIT_HOOKS_DIR` | The directory to use for git hooks, if required. | `/opt/kie/data/git/hooks` | False
 |`DECISION_CENTRAL_VOLUME_CAPACITY` | -- | Size of the persistent storage for Decision Central's runtime data. | 1Gi | True
-|`DECISION_CENTRAL_MEMORY_LIMIT` | -- | Decision Central Container memory limit. | 2Gi | False
-|`KIE_SERVER_MEMORY_LIMIT` | -- | KIE server Container memory limit. | 1Gi | False
+|`DECISION_CENTRAL_MEMORY_LIMIT` | -- | Decision Central Container memory limit. | 8Gi | True
+|`DECISION_CENTRAL_JAVA_MAX_MEM_RATIO` | `JAVA_MAX_MEM_RATIO` | Decision Central Container JVM max memory ratio. -Xmx is set to a ratio of the container available memory. The default is 80 which means 80% of the available memory is used as an upper boundary. You can skip this mechanism by setting this value to 0 in which case no -Xmx option is added. | 80 | True
+|`DECISION_CENTRAL_CPU_LIMIT` | -- | Decision Central Container cpu limit. | 2000m | True
```

Comment:
CPU

Reply:
again - autogenerated content. And I did not have a chance to edit the template as this was a last-moment addition.